examples/bank_account/bank_account.py

```diff
@@ -16,7 +16,7 @@ def info(self):
     def deposit(self, amount):
         """ deposit money """
         if amount > 0:
-            self._balance += amount - self._calc_commission_rate(self._hasCommissionDiscount)
+            self._balance -= amount - self._calc_commission_rate(self._hasCommissionDiscount)
```

Comment:
The deposit method should add the amount to the balance, not subtract it. Please revert the change. [important]